### PR TITLE
let script know if its deployment is still active

### PR DESCRIPTION
**Pull request by itayd**
`ak.is_deployment_active()`

if it's not, it must be draining.

### CODE REVIEW

File: internal/backend/akmodules/ak/module.go
[the remark this answers is not in the document]
Comment (itayd) [reply internal/backend/akmodules/ak/module.go:1]: ENG-1023

File: tests/system/parse.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var (
 	// wait <duration> for session <session ID>
 	waitAction = regexp.MustCompile(`^wait\s+(.+)\s+(for|unless)\s+session\s+(.+)`)
 
-	// output <equals|equals_json|contains|regex> [file] *
+	// output <equals|equals_json|equals_contains|regex> [file] *
```
Comment (daabr) [inline tests/system/parse.go]: Is this a mistake? I don't see `equals_contains` anywhere else

**Review by daabr — APPROVED** [review]

File: tests/system/parse.go (hunk 1 above)
Comment (itayd) [inline tests/system/parse.go]:
```suggestion
	// output <equals|equals_json|equals|regex> [file] *
```

File: tests/system/parse.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ var (
 	// wait <duration> for session <session ID>
 	waitAction = regexp.MustCompile(`^wait\s+(.+)\s+(for|unless)\s+session\s+(.+)`)
 
-	// output <equals|equals_json|contains|regex> [file] *
+	// output <equals|equals_json|equals|regex> [file] *
```
Comment (itayd) [inline tests/system/parse.go]:
```suggestion
	// output <equals|equals_json|contains|regex> [file] *
```

File: tests/system/parse.go (hunk 1 above)
Comment (itayd) [reply tests/system/parse.go]: oops

**Pull request merged**